pkg/apis/workspaces/v1alpha2/commands.go

```diff
@@ -58,7 +58,11 @@ type Command struct {
 	// Mandatory identifier that allows referencing
 	// this command in composite commands, from
 	// a parent, or in events.
-	Id           string `json:"id"`
+	Id string `json:"id"`
+	// Map of implementation-dependant free-form YAML attributes.
+	// +optional
+	// +devfile:overrides:include:omit=true
+	Attributes   attributes.Attributes `json:"attributes,omitempty"`
```

Comment:
Is this formatted? I don't know if field type should align with struct tags below

Reply:
It is yes. That's probably a bug of the formatter itself.